examples/mp-metrics-histogram/README_pt.adoc

```diff
@@ -0,0 +1,452 @@
+    # TYPE application:temperatures_degrees F histogram
+    application:temperatures_degrees F{quantile="0.98"} 54.0
+    # TYPE application:temperatures_degrees F histogram
+    application:temperatures_degrees F{quantile="0.99"} 54.0
+    # TYPE application:temperatures_degrees F histogram
+    application:temperatures_degrees F{quantile="0.999"} 54.0
+    # TYPE application:org_superbiz_histogram_weather_service_temperatures summary histogram
+    # TYPE application:org_superbiz_histogram_weather_service_temperatures_count histogram
+    application:org_superbiz_histogram_weather_service_temperatures_count 0.0
+    # TYPE application:org_superbiz_histogram_weather_service_temperatures_min histogram
+    application:org_superbiz_histogram_weather_service_temperatures_min 0.0
+    # TYPE application:org_superbiz_histogram_weather_service_temperatures_max histogram
+    application:org_superbiz_histogram_weather_service_temperatures_max 0.0
+    # TYPE application:org_superbiz_histogram_weather_service_temperatures_mean histogram
+    application:org_superbiz_histogram_weather_service_temperatures_mean NaN
+    # TYPE application:org_superbiz_histogram_weather_service_temperatures_stddev histogram
+    application:org_superbiz_histogram_weather_service_temperatures_stddev 0.0
+    # TYPE application:org_superbiz_histogram_weather_service_temperatures histogram
+    application:org_superbiz_histogram_weather_service_temperatures{quantile="0.5"} 0.0
+    # TYPE application:org_superbiz_histogram_weather_service_temperatures histogram
+    application:org_superbiz_histogram_weather_service_temperatures{quantile="0.75"} 0.0
+    # TYPE application:org_superbiz_histogram_weather_service_temperatures histogram
+    application:org_superbiz_histogram_weather_service_temperatures{quantile="0.95"} 0.0
+    # TYPE application:org_superbiz_histogram_weather_service_temperatures histogram
+    application:org_superbiz_histogram_weather_service_temperatures{quantile="0.98"} 0.0
+    # TYPE application:org_superbiz_histogram_weather_service_temperatures histogram
+    application:org_superbiz_histogram_weather_service_temperatures{quantile="0.99"} 0.0
+    # TYPE application:org_superbiz_histogram_weather_service_temperatures histogram
+    application:org_superbiz_histogram_weather_service_temperatures{quantile="0.999"} 0.0
+----
+
+=== Formato JSON:
+
+[source,bash]
+----
+$ curl -X GET http://localhost:8080/mp-metrics-histogram/metrics/application
+----
+
+=== Resposta JSON:
+
+[source,javascript]
+----
+{
+    "org.superbiz.histogram.WeatherService.temperatures": {
+        "count":0,
+        "max":0,
+        "min":0,
+        "p50":0.0,
+        "p75":0.0,
+        "p95":0.0,
+        "p98":0.0,
+        "p99":0.0,
+        "p999":0.0,
+        "stddev":0.0,
+        "unit":"none"
+    }
+}
+----
+
+== Metadados da métrica:
+
+Uma métrica terá metadados para que você possa aprender mais sobre ela, como `displayName`,`descrição`, `tags` etc.
```

Comment:
Here need translate descrição? Because the words 'displayName' and 'tags' are not translated. I saw this spanish text without translation for this words.

My suggestion is change to:
Uma métrica terá metadados para fornecer informação sobre ela como displayName, description, tags, etc.